lightning-background-processor/src/lib.rs

```diff
@@ -324,6 +336,13 @@ impl BackgroundProcessor {
 		self.stop_and_join_thread()
 	}
 
+	/// Signal to `BackgroundProcessor` that graph sync has completed
+	/// This function can only be called usefully once, so there is an implicit understanding
+	/// that running graph sync multiple times after startup is API misuse.
```

Comment:
This is confusingly worded, I think - if a client application stays open for a few hours, they should totally keep their graph up to date whereas this reads to me as "you should only sync the graph once at start". Maybe we should name the function `initial_graph_sync_complete` or so?